### PR TITLE
Expose Machine Fingerprint in PAL

### PR DESCRIPTION
**Pull request by davidkelly**
For now, just putting the machine fingerprint UUID in the PAL where the domain-server's uuid was shown before. 

Test Plan:
Open up PAL with another person (running this PR) in the domain.  You will see the uuid below their name be their machine fingerprint, when they are anonymous (since only anonymous users send machine fingerprint).   That fingerprint is easy to see in the interface log -- right at the top maybe 2nd line down.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 36792e005a88ac0898f7b6936f65f0bf9b7d531c available at: https://gist.github.com/e493f957b80b12db418c98fcbb7a239b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-36792e0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-36792e0.dmg

File: libraries/networking/src/udt/PacketHeaders.h [hunk 1]
```diff
@@ -241,4 +241,8 @@ enum class AudioVersion : PacketVersion {
     SpaceBubbleChanges,
 };
 
+enum class UsernameFromIDReplyVersion : PacketVersion {
```
Comment (howard-stearns) [inline libraries/networking/src/udt/PacketHeaders.h]: I thought we had putting the domain info listing changes under enum class DomainListVersion? (See https://github.com/highfidelity/hifi/pull/9226/files.)

Comment (davidkelly) [inline libraries/networking/src/udt/PacketHeaders.h]: Oh -- true.  I'd just versioned the specific message that changed.  Is there a reason not to do that?  I'm fine with just adding a new enum to that, but it seems less specific.  Fine with me though.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5b9c19b39e98ee43b2c0bc224d79f8a24dd3fa5e available at: https://gist.github.com/e493f957b80b12db418c98fcbb7a239b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-5b9c19b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-5b9c19b.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4feec85337de18b98b763deb3b5404848710b397 available at: https://gist.github.com/e493f957b80b12db418c98fcbb7a239b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-4feec85.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9235-4feec85.dmg

**Comment by howard-stearns** [comment]
When I am not logged in, I see the machine fingerprint of other users in the PAL. What's going on there? There's not supposed to be any way for non-admins to track people from one session to another.  That would be a QA fail if I'm interpreting correctly.

**Comment by howard-stearns** [comment]
It's ok. The problem was that I was testing on localhost, which has kick rights regardless of login. I have confirmted that when localhost has no kick rights, and user howard.stearns does have kick rights, then I see the info while logged in as howard.stearns, and do NOT see the information when I log out (and toggle the PAL).  QA Pass.

**Pull request merged**